external-crates/move/crates/move-compiler/tests/move_check/parser/spec_parsing_old_fun_fail.move

```diff
@@ -5,7 +5,7 @@ module 0x8675309::M {
     spec fun with_aborts_if {
```

Comment:
This whole file just got very strange. Can we do anything to help it?

Reply:
Didn't @tzakian do something that basically eats all the tokens after a `spec`? Do we just need to make that ever so slightly more clever?

Reply:
Yep! The `consume_spec_string` function does exactly that -- will consume the whole thing including everything before the curlies too.

Reply:
Fixed. What we were doing was to return after encountering old syntax rather than continuing to eat the spec